bindings/matlab/Visualization/visualizeRobot.m

```diff
@@ -0,0 +1,56 @@
+% This script is an example on how to use the matlab Idyntree visualization
+
+% insert the path where the meshes may be found, it repaces what is require
+% d to complete the path inside the urdf visual mesh field.
+meshFilePrefix=[getenv('ROBOTOLOGY_SUPERBUILD_SOURCE_DIR') '/build-linux/install/share'];
```

Comment:
See https://github.com/robotology/idyntree/pull/668/files#r403447581 . If this file is meant to be used in practice, the meshFilePrefix should be some kind of external argument.

Reply:
Is not meant to be used in practice is just like a tutorial or example on how to use the visualization.

Reply:
moved to examples/iDynTreeWrappers , still need to figure out the meshes for the testing

Reply:
You can also document that you need to manually set the prefix to the location where icub-models is install.

Reply:
I wrote what the meshPrefix is and why is needed in the documentation I added in docs and inside the perpareVisualization and getMeshes functions.

Reply:
Done